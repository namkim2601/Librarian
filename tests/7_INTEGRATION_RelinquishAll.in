ReLiNqUiSh AlL nomembers
add member Nam Kim
relinquish all nosuchmember
relinquish all 100000
exit
