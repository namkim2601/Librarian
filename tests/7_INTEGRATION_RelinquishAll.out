user: No members in system.

user: Success.

user: No such member in system.

user: Success.

user: Ending Library process.
